frontend/src/pages/Login.jsx

```diff
@@ -72,11 +163,10 @@ const LoginPage = () => {
                   </a>
                   </div>
 
-                  <div className="flex w-full mt-4 justify-center items-center px-5">
-                     {/*TODO: change this to proper link */}
-                      <a href="/home" className="btn btn-link btn-accent w-1/2 text-center">
-                          Reset Password
-                      </a>
+                  <div className="w-full mt-4 justify-center items-center px-5">
+                    <div className="bg-white text-center text-darkmint border border-darkmint py-2 rounded hover:bg-lightmint">
+                      <button onClick={handleForgot}>Reset password</button>
```

Comment:
Do we want this to be a button or a clickable link? Also, make sure to use daisy-UI components for consistency